app/db/models/fund.py

```diff
@@ -19,6 +21,23 @@
 BaseModel: DefaultMeta = db.Model
 
 
+class FundingType(Enum):
+    COMPETITIVE = "COMPETITIVE"
+    UNCOMPETED = "UNCOMPETED"
+    EOI = "EOI"
+
+    def get_text_for_display(self):
+        match self:
+            case FundingType.COMPETITIVE:
+                return "Competitive"
+            case FundingType.EOI:
+                return "EOI"
+            case FundingType.UNCOMPETED:
+                return "Uncompeted"
+            case _:
+                return self.value
```

Comment:
if any other type given don't we need to throw an error ?

Reply:
No I don't think so - this is more a catch all incase we update the enum but forget to update this function.